recipe/meta.yaml

```diff
@@ -10,20 +9,21 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  {{ hash_type }}: {{ hash }}
+  sha256: {{ hash }}
 
 build:
   noarch: python
-  number: 1
-  script: python -m pip install --no-deps --ignore-installed .
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
     - pip
     - python
+    - python >=3.4
```

Comment:
Conda wants a minimum version. Since conda does not support 2.7, I set it to 3.4+. Is that right?

Reply:
I think you should remove this line.

Reply:
Actually, I'm wrong 😄!  You should actually constrain it on both the host and run sections.  So, you should remove line 22 and add something like line 23 (may be `python >=3` is more adequate?).  Then, you should also do that on the `run` section.  Instructions from here: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python